### PR TITLE
:up: Update ES image to be one with latest mappings and data

### PR DESCRIPTION
**Pull request by st3v3nhunt**
Use the latest image for [`profiles-db-elastic`](https://hub.docker.com/r/nhsuk/profiles-db-elastic/tags/) i.e. `0.4.1-20170930`.
This image has the latest set of mappings and data added to the ES instance via the etl combiner and updater jobs. The changes required have been to update the data in accordance with that in the image.

### CODE REVIEW

File: test/integration/resultsPage.js [hunk 1]
```diff
@@ -140,46 +140,31 @@ describe('Results page', () => {
...
-        expect(searchResults.text()).to.equal('Crookes Valley Medical Centre');
-        expect(searchResults.attr('href')).to.equal('https://systmonline.tpp-uk.com/Login?PracticeId=C88057');
+        expect(searchResults.text()).to.equal('Medi Access Ltd');
+        expect(searchResults.attr('href')).to.equal('https://systmonline.tpp-uk.com/Login?PracticeId=B81693');
       });
     });
   });
 
   describe('Surgeries with booking system which can not be linked to', () => {
-    describe('when the surgery has a website', () => {
```
Comment (st3v3nhunt) [inline test/integration/resultsPage.js]: I can't see another surgery where this condition is true so I've removed the test.

Comment (markysoft) [reply test/integration/resultsPage.js]: How about 'Sunlight Centre Surgery'?

Comment (st3v3nhunt) [reply test/integration/resultsPage.js]: Great suggestion. How did you find it?

**Comment by st3v3nhunt** [comment]
Used the suggestion of `Sunlight Centre Surgery` to get all tests covered.
Also updated the packages and removed the snyk policy file as it is no longer needed, as in there are no vulnerabilities needing to be ignored.

**Pull request merged**